### ISSUE TITLE
Bug: Updown Widget Negative Number w/ Min = 0

### ISSUE THREAD

**Issue by ClockerZadq** [mozilla-services/react-jsonschema-form#347]
### Prerequisites

- [x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme).

### Description

The widget "updown" for type number allows scrolling into the negative numbers when the minimum is set to zero. The validation will fail exclaiming that the minimum is 0. If the minimum is set to 1, the widget behaves as expected.

### Steps to Reproduce

This is the schema that I am using:

```js
{  
  "PocketLint":{  
    "type":"number",
    "default":50,
    "widget":"updown",
    "multipleof":1,
    "visibility":{  
      "hidden":false
    },
    "randomize":false,
    "validate":{  
      "min":0,
      "max":99
    },
    "help":"",
    "title":"Pocket Lint",
    "id":"PocketLint"
  }
}
```

Render the widget
Hold the down arrow

#### Expected behavior

The data on the field will stop at zero

#### Actual behavior

The data of the field continues past zero into the negative numbers.

### Version

0.40.0

**Comment by n1k0** [comment]
This has just been fixed in #344. Please wait for an upcoming new release.

**Comment by ClockerZadq** [comment]
Perfect! Cheers
